src/PKSim.Core/Model/IndividualParameterSameFormulaOrValueForAllSpecies.cs

```diff
@@ -11,5 +7,6 @@ public class IndividualParameterSameFormulaOrValueForAllSpecies
       public string ParameterName { get; set; }
       public bool IsSameFormula { get; set; }
       public bool IsSameValue => !IsSameFormula;
+      public bool IsSame => IsSameFormula || IsSameValue;
```

Comment:
often I just want to know is it the same value or formula. I don't really care for the specific. If I export, it will be exported based on the constant vs formula anyways